src/pages/MovieView.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import MovieList from '../components/MovieList/MovieList';
+import SearchBar from '../components/SearchBar/SearchBar';
+import { useMovies } from '../contexts/MovieProvider';
+import { palette } from '../assets/stylesConstants';
+
+export default function MovieView() {
+  const { setTitle } = useMovies();
+
+  return (
+    <div css={PageStyle}>
+      <SearchBar
+        onSubmit={async ({ search }) => {
+          await setTitle(search);
+        }}
```

Comment:
setTitle 은 비동기 코드가 아니므로, async 키워드는 제거해주셔도 괜찮을 것 같아요!
```suggestion
        onSubmit={({ search }) => {
          setTitle(search);
        }}
```